contracts/ExitHandler.sol

```diff
@@ -181,6 +181,64 @@ contract ExitHandler is DepositHandler {
     exitDuration = _exitDuration;
   }
 
+  function startLimboExit(
+    bytes memory limboTxData, bytes32 signature, uint8 _outputIndex,
+    bytes memory depositTxData, bytes memory depositProof
+  ) public payable {
+    // Transaction is from A to B
+    // B will have to pay exitStake 
+    require(msg.value >= exitStake, "Not enough ether sent to pay for exit stake");
+    TxLib.Tx memory transferTx = TxLib.parseTx(limboTxData);
+    TxLib.Output memory out = transferTx.outs[_outputIndex];
+    
+    // check if B has the right to do the exit for A or not
+    bytes32 sigHash = TxLib.getSigHash(limboTxData);
+    require(sigHash == signature);
+
+
+    // isWellDecodedTransaction();
+    bytes32 limboTxHash = keccak256(limboTxData);
+
+    bytes32 utxoId = bytes32(uint256(_outputIndex) << 120 | uint120(uint256(limboTxHash)));
+    uint256 priority;
+
+    require(depositProof.length > 0, "Not a limbo exit");
+    
+    // check youngest input tx inclusion in the root chain block
+    bytes32 depositTxHash;
+    (txPos, depositTxHash,) = TxLib.validateProof(96, depositProof);
+    require(
+      depositTxHash == transferTx.ins[*].outpoint.hash, 
```

Comment:
Was supposed to pass _inputIndex  in the function and put it here but i was looking for some other way to verify this.